src/shared/inc/gcinfodecoder.h

```diff
@@ -222,7 +226,7 @@ enum GcInfoDecoderFlags
     DECODE_PROLOG_LENGTH         = 0x400,   // length of the prolog (used to avoid reporting generics context)
     DECODE_EDIT_AND_CONTINUE     = 0x800,
     DECODE_REVERSE_PINVOKE_VAR   = 0x1000,
-    DECODE_RETURN_KIND           = 0x2000,
+    DECODE_RETURN_KIND           = 0x2000,  // not currently used, but SOS needs this for older versions
```

Comment:
```suggestion
#ifdef DECODE_OLD_FORMATS
    DECODE_RETURN_KIND           = 0x2000,  // Used starting with v3 format
#endif
```

Reply:
This was probably meant "Unused starting with v4 format"

Reply:
Yes, you are right